plugins/inputs/gnmi/handler.go

```diff
@@ -62,6 +64,10 @@ func (h *handler) subscribeGNMI(ctx context.Context, acc telegraf.Accumulator, t
 		))
 	}
 
+	if h.ClientParameters.Time > 0 {
+		opts = append(opts, grpc.WithKeepaliveParams(h.ClientParameters))
+	}
```

Comment:
Given the comment in the readme, won't this always be set? i.e. time has a minimum of 10? Does this need to be wrapped around the if statement?

Reply:
Maybe we need to tweak the wording in the readme.  The default in the library is infinite (i.e. it's unset); here the `if` statement checks to see if the user has set any value in the `keepalive_time` config, which gets copied to the struct; the library will override any value less than `10s` to be `10s`, from what I understand.

Reply:
> Maybe we need to tweak the wording in the readme.

Previously we were never using the struct, so I can see it being set to infinite. However, now you are always initializing it. Doesn't that mean 10s will always be set going forward?

Reply:
I've tweaked the docs; hopefully that is clearer.  I'm open to an alternative.

Reply:
I think that tweak definitely helps. The config should show the default value, so in this case, I suggest leaving it empty if it is not used when not set.

Reply:
Okay I see that it's common to just use the empty string, updated.  Please take another look.